src/components/Sidebar/contentTree.js

```diff
@@ -123,13 +123,23 @@ const ContentTree = ({ edges, location }) => {
   const linkAs = props => {
     // Ref: https://codesandbox.io/s/93olj3k1xr?file=/src/component/ContextMenuNav.js:2431-2435
     return (
-      <Link to={props.link.url} {...props} >
+      <Link  id={props.link.key} to={props.link.url} {...props} >
         {props.link.name}
       </Link>
     );
 
   }
 
+  const handleLinkClick = () => {
+    const element = document.getElementById('scroll-to-top-main-content');
+    element?.scrollTo(0,0)
+  };
+
+  useEffect(()=> {
+    const sidebarElements = document.getElementsByClassName('ms-Nav-compositeLink is-selected');
+    if(sidebarElements.length > 0) sidebarElements[0].scrollIntoView({block: "center", inline: "nearest"});
```

Comment:
Just a good practice, for `if` statement, if it is not early return, we should prefer if + curry braces. 
It is safer if someone add more statements if an if block return true.
For example.
```ts
if(sidebarElements.length > 0) {
   sidebarElements[0].scrollIntoView({block: "center", inline: "nearest"});
   // more statements in future are safe.
}
```

Reply:
OK, I  see